README.md

```diff
@@ -1,5 +1,8 @@
 # xk6-sm
 
+> [!WARNING]
+> Heads up, Synthetic Monitoring users: You do not need to build or download this. A k6 binary compiled with this extension is already shipped in our [agent](https://github.com/grafana/synthetic-monitoring-agent) packages.
```

Comment:
The tone of the message is unnecessarily adversarial.

You can get the same result by simply stating that the code found in this repo is delivered as part of the packages that are available for download as part of the agent releases.